OasisDEX/oasis-borrow
# Extracted add / remove trigger control, adjusted stop loss flow

# [Extracted add / remove trigger control](https://app.shortcut.com/oazo-apps/story/5834/control-for-removing-triggers-which-supports-all-auto-features)

# [Adjusted stop loss flow to other auto flows](https://app.shortcut.com/oazo-apps/story/5833/stop-loss-refactor-to-align-with-other-auto-features)

<please insert a shortcut link above>
  
## Changes 👷‍♀️
  <Please add short list of changes>

- extracted add / remove trigger control which wraps sidebars and provide button handlers
- all auto features use remove method from aggregator to remove triggers
- refactored stop loss, single sidebar and reducer for add & remove trigger
- fixed issues with default slider state
- fixed issues with isEditing state
  
## How to test 🧪
  <Please explain how to test your changes>

- add & remove all possible auto features in different combinations, make sure that all middle steps works as expected and there is no issue with transactions
